File: 1470-shuffle-the-array/1470-shuffle-the-array.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        int i=0,chk=n;
        vector<int> v;
        while(i<n) {
            v.push_back(nums[i++]);
            v.push_back(nums[chk++]);
        }
        return v;
    }
};

File: 1470-shuffle-the-array/1470-shuffle-the-array.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        i=0
        chk=n
        ans=[]
        while(i<n):
            ans.append(nums[i])
            ans.append(nums[chk])
            i+=1
            chk+=1
        return ans